portality/bll/exceptions.py

```diff
@@ -42,4 +44,36 @@ class ConfigurationException(Exception):
     """
     Exception to raise when our own configuration is broken
     """
-    pass
\ No newline at end of file
+    pass
+
+class DuplicateArticleException(Exception):
+    """
+    Exception to raise when a duplicate article is detected, and this is not permitted
+    """
+    pass
+
+class IngestException(Exception):
```

Comment:
should this be a more generalised traceable exception that we can use elsewhere?

Reply:
as discussed, maybe but not now.